pkg/portal/regions_test.go

```diff
@@ -0,0 +1,221 @@
+package portal
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"encoding/json"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/go-test/deep"
+	"github.com/gorilla/mux"
+
+	testdatabase "github.com/Azure/ARO-RP/test/database"
+)
+
+func TestRegionList(t *testing.T) {
```

Comment:
Need a test for behavior when `AZURE_ENVIRONMENT` is not the same as `azureclient.PublicCloud.Environment.Name`. Returning an empty list is fine as long as the frontend component knows how to deal with it.

Reply:
Have added two clear tests for when the environment is set to public and when it is set to Fairfax